new-pipeline/dist.html

```diff
@@ -23,185 +23,266 @@
         <header>
             <form class="navbar-form inline pull-right">
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn btn-success" title="Report a bug in Telemetry dashboards" target="_blank">Report bug</a>
-                <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view">Evolution View</a>
+                <!-- <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view">Evolution View</a> -->
```

Comment:
Leaving commented-out code in the tree is unnecessary (we make one exception for warning notices, so we know where to put them in a hurry)